packages/core/src/components/DependencyGraph/DefaultLabel.tsx

```diff
@@ -0,0 +1,36 @@
+/*
+ * Copyright 2020 Spotify AB
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import React from 'react';
+import { makeStyles } from '@material-ui/core/styles';
+import { BackstageTheme } from '@backstage/theme';
+import { RenderLabelProps } from './types';
+
+const useStyles = makeStyles((theme: BackstageTheme) => ({
+  text: {
+    fill: theme.palette.textContrast,
+  },
+}));
+
+function DefaultLabel({ edge: { label } }: RenderLabelProps) {
+  const classes = useStyles();
+  return (
+    <text className={classes.text} textAnchor="middle">
+      {label}
+    </text>
+  );
+}
+export default DefaultLabel;
```

Comment:
Hi! We have decided to pretty strongly [avoid default exports](https://github.com/backstage/backstage/blob/master/docs/architecture-decisions/adr003-avoid-default-exports.md) - please see rationale in that article. I know it's not a fun ask, but would you mind changing this and all below to be named exports instead?

Reply:
Of course!